build.gradle.kts

```
@@ -69,6 +72,12 @@ tasks.validatePlugins {
     enableStricterValidation.set(true)
 }
 
+tasks.matching {
+    it is HonkerCheckTask || it is HonkerGenDependenciesTask || it is HonkerGenLicenseTask || it is HonkerGenNoticeTask
+}.configureEach {
+    notCompatibleWithConfigurationCache("https://github.com/eskatos/honker-gradle/issues/1")
+}
+
```

Let's avoid using `matching {}` here as it defeats task configuration avoidance.
```suggestion
listOf(
    HonkerCheckTask::class,
    HonkerGenDependenciesTask::class,
    HonkerGenLicenseTask::class,
    HonkerGenNoticeTask::class
).forEach { honkerTaskType ->
    tasks.withType(honkerTaskType).configureEach {
        notCompatibleWithConfigurationCache("https://github.com/eskatos/honker-gradle/issues/1")
    }
}

```